updater/defaults/defaults.go

```diff
@@ -63,6 +64,7 @@ func inner(ctx context.Context) error {
 	updater.Register("debian", df)
 
 	updater.Register("osv", new(osv.Factory))
+	updater.Register("rhel-vex", new(vex.Factory))
```

Comment:
Is the idea to support both OVAL and VEX at the same time for a period, or will this PR remove `"rhel"`, too, later?

Reply:
Probably remove the OVAL in the same PR

Reply:
@RTann check out the last commit

Reply:
yep, arbitrary afaik